aws_lambda_builders/path_resolver.py

```diff
@@ -0,0 +1,21 @@
+from aws_lambda_builders.utils import which
+
+
+class PathResolver(object):
+
+    def __init__(self, language, runtime):
+        self.language = language
+        self.runtime = runtime
+        self.executables = [self.runtime, self.language]
```

Comment:
True it is inspired from python's packages (tox, pipenv). My thinking is that the lambda runtime needs to be prioritized so that we can create arbitrary mapping via symlinks.